src/Components/Search/NewSearch/NewSuggestionItem.tsx

```diff
@@ -0,0 +1,173 @@
+      <QuickNavigation option={option} />
+    </SuggestionItemLink>
+  )
+}
+
+interface DefaultSuggestionProps {
+  option: SuggionItemOptionProps
+  query: string
+}
+
+const DefaultSuggestion: FC<DefaultSuggestionProps> = ({ option, query }) => {
+  const matches = match(option.text, query)
+  const parts = parse(option.text, matches)
+  const partTags = parts.map(({ highlight, text }, index) =>
+    highlight ? <Highlight key={index}>{text}</Highlight> : text
+  )
+
+  return (
+    <Flex alignItems="center">
+      <SuggestionItemPreview
+        imageUrl={option.imageUrl}
+        label={option.subtitle}
+      />
+      <Spacer x={1} />
+      <Flex flexDirection="column" flex={1} overflow="hidden">
+        <Text variant="sm-display" overflowEllipsis>
+          {partTags}
+        </Text>
+
+        <Text color="black60" variant="xs" overflowEllipsis>
+          {option.subtitle}
+        </Text>
+      </Flex>
+    </Flex>
+  )
+}
+
+interface QuickNavigationProps {
+  option: SuggionItemOptionProps
+}
+
+const QuickNavigation: FC<QuickNavigationProps> = ({ option }) => {
+  const { trackEvent } = useTracking()
+
+  const handleArtworksItemClicked = () => {
+    trackEvent(
+      tracks.quickNavigationItemClicked({
+        destinationPath: `${option.href}/works-for-sale`,
+        label: "Artworks",
+      })
+    )
+  }
+
+  const handleAuctionResultsItemClicked = () => {
+    trackEvent(
+      tracks.quickNavigationItemClicked({
+        destinationPath: `${option.href}/auction-results`,
+        label: "Auction Results",
+      })
+    )
+  }
+
+  if (!option.showArtworksButton && !option.showAuctionResultsButton)
+    return null
+
+  return (
+    <Flex flexWrap="wrap">
+      {!!option.showArtworksButton && (
+        <QuickNavigationItem
+          onClick={handleArtworksItemClicked}
+          to={`${option.href}/works-for-sale`}
+          Icon={ArtworkIcon}
+        >
+          Artworks
+        </QuickNavigationItem>
+      )}
+      {!!option.showAuctionResultsButton && (
+        <QuickNavigationItem
+          onClick={handleAuctionResultsItemClicked}
+          to={`${option.href}/auction-results`}
+          Icon={GavelIcon}
+        >
+          Auction Results
+        </QuickNavigationItem>
+      )}
+    </Flex>
+  )
+}
+
+interface QuickNavigationItemProps {
+  to: string
+  onClick?: (event: React.MouseEvent<HTMLAnchorElement>) => void
+}
+
+const QuickNavigationItem: FC<QuickNavigationItemProps & PillProps> = ({
+  to,
+  onClick,
+  ...rest
+}) => {
+  const handleClick = event => {
+    // Stopping the event from propagating to prevent SearchBar from navigation to the main suggestion item url.
+    event.stopPropagation()
+    event.preventDefault()
+    onClick?.(event)
+
+    // TODO: Use routerLink instead of manual navigation?
+    // FIXME: Using `window.location.assign(to)` instead of `router.push(to)` to prevent a bug where the search bar won't hide anymore.
+    window.location.assign(to)
```

Comment:
**suggestion**: as far as I understand we can use `router.push` now without any problems

https://user-images.githubusercontent.com/3513494/236444119-2f858c6e-4aab-4d73-b4c5-c2f2b1b47359.mp4